README.md

```diff
@@ -1149,6 +1154,61 @@ player.setVolume(0.5).then(function(volume) {
 });
 ```
 
+### getQualities(): Promise&lt;object[], Error&gt;
+
+Get the available qualities of the current video.
+
+```js
+player.getQualities().then(function(qualities) {
+    // qualities = an array of quality objects
+}).catch(function(error) {
+    // an error occurred
+});
+```
+
+Each quality object looks like this:
+
+```js
+{
+    "label": "4K",
+    "id": "2160p",
+    "active": true
+}
+```
+
+### getQuality(): Promise&lt;string, Error&gt;
+
+Get the current selected quality.
+
+```js
+player.getQuality().then(function(quality) {
+    // quality = the current selected quality
+}).catch(function(error) {
+    // an error occurred
+});
+```
+
+### setQuality(quality: string): Promise&lt;string, (TypeError|Error)&gt;
+
+Set the quality of the video. (available to Plus, PRO and Business accounts)
+
+```js
+player.setQuality('720p').then(function(quality) {
```

Comment:
Question - is there a quality setting for 'auto'?  I'd guess people would want to try higher quality settings but then switch back to 'auto' if they're not working as well as expected.  I'm guessing 'auto' is one quality getQuality() will return?  Thanks for getting this done!

Reply:
good question, yup `auto` will be one of the items returned by `getQualities()`

Reply:
I'm seeing an issue on this @luwes where I don't get the quality 'auto' on mobile but I do get it on desktop.  Trying to setQuality('auto') also fails on mobile but not desktop even though by default the current quality on mobile is 'auto'